app/helpers/form_helper.rb

```diff
@@ -9,9 +9,6 @@ def us_states_territories
       ['American Samoa', 'AS'],
       ['Arizona', 'AZ'],
       ['Arkansas', 'AR'],
-      ['Armed Forces Americas', 'AA'],
```

Comment:
This method is used for more than just in-person proofing, but the ticket seems to be focused at in-person proofing. To avoid side-effects, I think we'll want to filter where we're using it in in-person proofing, rather than removing the state options everywhere.

Reply:
Ah good catch thanks! So those options don't result in errors on the non-ipp side of things?

Reply:
If so, maybe we should get USPS to update the API to be able to handle them in all cases and then not remove them at all since the error would no longer happen?

Reply:
It would be ideal if USPS could handle the values, but depending on turnaround time for that to happen, it's probably easier for us to implement a workaround on our end.

Do we know if they support an equivalent option for these, so that we could at least support users who would otherwise use them?

Reply:
Hmm the API spec they gave us doesn't seem to have any notes about it. I can write up a draft email to them asking what the turn around may be/if they want to support those options then we will have better direction at the start of next week.

Reply:
Maybe the easiest path forward: 

1. Write a filter to remove items from the in person proofing views to get this ticket in
2. Email USPS asking if it possible and estimated time. We can then write up a new ticket to delete filters and put it on the sprint when USPS expects to be done if this is something they will do.  I think better to get this ticket complete rather than keeping it open. My guess with communication and turn around time that it will be at least 1-2 sprints.

Reply:
Good call, forgot you mentioned their turn around time was fairly long 😬